src/toil/batchSystems/torque.py

```diff
@@ -113,10 +117,20 @@ def prepareQsub(self, cpu, mem, jobID):
             reqline = list()
             if mem is not None:
                 memStr = str(mem / 1024) + 'K'
-                reqline += ['-l mem=' + memStr]
+                reqline.append('mem=' + memStr)
 
             if cpu is not None and math.ceil(cpu) > 1:
-                qsubline.extend(['-l ncpus=' + str(int(math.ceil(cpu)))])
+                reqline.append('ncpus=' + str(int(math.ceil(cpu))))
+
+            # Other resource requirements can be passed through the environment (see man qsub)
+            reqlineEnv = os.getenv('TOIL_TORQUE_REQS')
```

Comment:
Can you add a brief section on this variable under "Environment variable options" in /toil/docs/running/running.rst?